app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from __future__ import annotations
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
```suggestion
    return sum(player.get_rating() for player in players)
```

![image](https://user-images.githubusercontent.com/104218550/214816157-d998f170-b7df-40e7-a48e-25eb19f68561.png)